orphedomos-maven-plugin/src/main/java/org/technologybrewery/orphedomos/mojo/DockerBuildMojo.java

```diff
@@ -30,6 +30,12 @@ public class DockerBuildMojo extends AbstractDockerMojo {
 
     private static final Logger logger = LoggerFactory.getLogger(DockerBuildMojo.class);
 
+    public void execute() throws MojoExecutionException, MojoFailureException {
+        setUpPlaceholderFileAsMavenArtifact();
```

Comment:
Couldn't we move this to one of the first steps in the `doExecute` method instead adding a new override so we can add it here?

Reply:
No, the way `skip` is implemented currently is in the `execute` method, so when `skip` is `true`, it never hits `doExecute` at all.

Reply:
Ah wait I think I understand what's going on.  Because skip is checked at the root level, `doExecute` is never called when skip is true.  Which presumably causes install step of an orphedomos module to fail, because there's no file to install.

Reply:
Exactly.